### PR TITLE
Removed react-native Share dependency

### LINKED ISSUE #585: ActionSheetIOS.showShareActionSheetWithOptions
## Ask your Question

Why use `ActionSheetIOS.showShareActionSheetWithOptions` on iOS when not using urls?
This currently crashes on iOS 13, but has been fixed in RNShare.open

### PR DESCRIPTION
**Pull request by Arjan-Zuidema**
# Overview
Fixes #585

# Test Plan
Tested in private project using real iPhone XR device and simulators

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -174,49 +173,34 @@ class RNShare {
...
-                  });
-                } else {
-                  reject(new Error('User did not share'));
-                }
-              },
-            );
+          if(options.url && !options.urls) // Backward compatibility with { Share } from react-native
+          {
+            const url = options.url;
+            options.url = undefined;
```
Comment (MateusAndrade) [inline index.js]: Applying this is broking the CI. What do you think about applying a ```delete options.url```?

Comment (Arjan-Zuidema) [reply index.js]: I was just on it :)

**Comment by tomtargosz** [comment]
Is this good to be merged in? iOS 13 is out so it'd be ideal if a new version with the fixes could be released.

**Comment by MateusAndrade** [comment]
@Arjan-Zuidema could you update this with the master version?

**Comment by Arjan-Zuidema** [comment]
Done, master has been merged

**Comment by MateusAndrade** [comment]
@Arjan-Zuidema sorry for the late reply. Tested this on both device and emulator and it works fine. Thanks for your help! :tada: :tada: :tada:

**Comment by MateusAndrade** [comment]
@jgcmarins i think we need a new release. :sweat_smile:

**Pull request merged**